### PR TITLE
Fix some simple typos

### PR DESCRIPTION
**Pull request by lpmi-13**
## Description
A few minor copy edits

## How to test
N/A

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2651"></a>

### CODE REVIEW

**Comment by lpmi-13** [comment]
It is not lost on me that my commit message itself contains a typo 😆

File: src/routes/blog/gitpod-installer.md [hunk 1]
```diff
@@ -61,11 +61,11 @@ We often merge the infrastructure and application concerns when we think of requ
...
 
 ## Technical differences from Helm charts
 
-While developing the Gitpod Installer, we took a call to be opinionated at times to build something that just works. This allows us to provide our experience of running Gitpod SaaS to owners of self-hosted implementations.. Being opinionated provides stability and keeps everything on a standards track. It reaps immense benefits while troubleshooting when something goes wrong. We enforced these decisions as validation constraints for the Gitpod Installer at config and cluster levels. These are some of the decisions we took,
+While developing the Gitpod Installer, we took a decision to be opinionated at times to build something that just works. This allows us to provide our experience of running Gitpod SaaS to owners of self-hosted implementations. Being opinionated provides stability and keeps everything on a standards track. It reaps immense benefits while troubleshooting when something goes wrong. We enforced these decisions as validation constraints for the Gitpod Installer at config and cluster levels. These are some of the decisions we took,
```
Comment (lpmi-13) [inline src/routes/blog/gitpod-installer.md:68]: Alternate suggestion for wording here could be "we made a call"

Comment (filiptronicek) [reply src/routes/blog/gitpod-installer.md:68]: My suggestion is to use `make` and `the` here, I think it sounds better. Alright with both alternatives as well though! 
```suggestion
While developing the Gitpod Installer, we made the decision to be opinionated at times to build something that just works. This allows us to provide our experience of running Gitpod SaaS to owners of self-hosted implementations. Being opinionated provides stability and keeps everything on a standards track. It reaps immense benefits while troubleshooting when something goes wrong. We enforced these decisions as validation constraints for the Gitpod Installer at config and cluster levels. These are some of the decisions we took,
```

**Comment by filiptronicek** [comment]
> It is not lost on me that my commit message itself contains a typo 😆

Nice 😆, fixed the PR title as well.

**Review by lucasvaltl — APPROVED** [review]
LGTM! :) Thank you

**Comment by lpmi-13** [comment]
I believe so. I've had a few other PRs merged into this repo, so that should already be sorted, unless the process has changed since March.

**Comment by meysholdt** [comment]
hi @lpmi-13, and thank you for your contribution! We don't have a CLA on record for you, unfortunately. Can you please sign our CLA via [this DocuSign form](https://eu.docusign.net/Member/PowerFormSigning.aspx?PowerFormId=30e1aa46-cce5-4bc6-a933-ad02cd8d5ada&env=eu&acct=86335c2c-01b9-4edc-a187-014f5150a112&v=2)? If there are any questions, you can reach me via moritz@gitpod.io.

**Comment by lpmi-13** [comment]
All signed now, so we should be gold

**Comment by lpmi-13** [comment]
I had been kind of expecting that to be one of the CI checks if it's a requirement. Is there a particular reason it's not?

**Comment by meysholdt** [comment]
hey @lpmi-13! Thank you for signing the CLA. 
It's not a CI check yet because there is still room for improvement regarding our automation :)

**Comment by lucasvaltl** [comment]
Merging this now - thank you very much for your contribution @lpmi-13!

**Pull request merged**